app/controllers/OverviewController.scala

```diff
@@ -220,7 +220,11 @@ class OverviewController @Inject()(
       reportType match {
         case PsrReportType.Sipp.name =>
           val sippUrl = s"${config.urls.sippBaseUrl}/${srn.value}${config.urls.sippStartJourney}"
-          Future.successful(Redirect(sippUrl))
+          Future.successful(
+            Redirect(sippUrl)
+              .addingToSession("taxYear" -> taxYear)
```

Comment:
Could be constant as e.g. `Constants.SUBMISSION_DATE`

Reply:
I can add as these constants if you wish 👍 Just let me know :)

Reply:
Already done please check and approve :)